Week3/homework/step3-bonus.js

```diff
@@ -5,6 +5,7 @@ const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 function makeUnique(arr) {
   // Replace this comment and the next line with your code
   console.log(arr);
+  return arr.filter((value, index) => arr.indexOf(value) === index);
```

Comment:
Nice solution! If you're interested I'd encourage you to check out `Set` which is another way to solve this one https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set